frontend/composables/useSession.ts

```diff
@@ -27,7 +25,8 @@ export const useSession = () => {
     // We're the first to request a user, kick of the request and hop in line at the front of the queue.
     return new Promise((resolve) => {
       resolvers.value.push(resolve)
-      void pactaClient.findUserByMe()
+      void usePACTA()
```

Comment:
You can't 👏use 👏composables 👏outside 👏of 👏`setup` 👏functions.

If you're trying to avoid the hit of having the whole composable be async, just embrace it. It needs to be async because `usePACTA` needs to be async because `useMSAL` needs to be async because we don't know if we're authenticated (at least on the client) until `useMSAL` initializes, which is an async action.